workos/_base_client.py

```diff
@@ -117,3 +113,15 @@ def user_management(self) -> UserManagementModule: ...
     @property
     @abstractmethod
     def webhooks(self) -> WebhooksModule: ...
+
+    @property
+    def base_url(self) -> str:
+        return self._base_url
```

Comment:
In base HTTP client we have a setter that enforces a trailing slash. Would be good to make sure these values are the same and perhaps enforce the trailing slash here instead:
https://github.com/workos/workos-python/blob/6d7ab8c39dce654c1a3fab3642df2533718da518/workos/utils/_base_http_client.py#L209-L216